content/blog/mlops-–-deploying-an-ml-model-in-greenlake-platform-mlops-service.md

```diff
@@ -0,0 +1,318 @@
+
+In this article, we focus on optimal model identified after data preparation and building model. More specifically, as an example, we shall deploy a sample ONNX model available in Kubernetes native MinIO Object storage, using Triton Inference Server.
+
+Triton Inference server supports deployment of any AI model from multiple deep learning and machine learning frameworks, including TensorRT, TensorFlow, PyTorch, ONNX, OpenVINO, Python, RAPIDS FIL, and more. Triton delivers optimized performance for many query types, including real time, batched, ensembles and audio/video streaming. To learn more about Triton Inference server, refer to the References section at the end of the post.
+
+HPE GreenLake for ML Ops platform allows customers to host their favorite cloud native applications say Mlflow, Minio, etc.
+
+**Pre-requisite**
+
+* An active service subscription to HPE GreenLake for ML Ops
+* ML Ops project has been created by ML Ops admin and user is able to launch to HPE Ezmeral Runtime Enterprise with ML Ops Project Member Role role
+* Access credentials to any S3 based object storage is available
+* Triton Inference Server container image accessible either through on-prem registry (eg Harbor) or public registry accessible.
+
+**Steps to deploy a model**
+
+1)   Validation connection to kubernetes cluster
+
+1. Click HPE GreenLake for ML Ops card in the HPE GreenLake Central **Dashboard** shows the number of previously created projects.
+
+   ![Select ML Ops Project](/img/blog_2.png "Select ML Ops Project")
+2. Click link “Launch ML Operations Console”
+3. Click “Dashboard” on left navigation of “Ezmeral Container Platform”
+4. Download “kubectl”, “HPE kubectl plugin”, and “kubeconfig”.
+5. Set environment variable KUBECONFIG to point to the kubeconfig file
+6. validate connectivity to the cluster using command “kubectl get no”
+
+2)   Place the model in s3 object storage
+
+1. Place the model and configuration file for triton inference server in object storage
+
+   ![object storage for model](/img/blog_1.png "object storage for model")
+2. Sample configuration file for the model as is below:
+
+```markdown
+name: "braintumor_onnx"
+
+platform: "onnxruntime_onnx"
+
+max_batch_size : 0
+
+input [
+
+  {
+
+    name: "input_1"
+
+    data_type: TYPE_FP32
+
+    dims: \[ 100, 128, 128, 2 ]
+
+  }
+
+]
+
+output [
+
+  {
+
+    name: "conv2d_22"
+
+    data_type: TYPE_FP32
+
+    dims: \[ -1, 128, 128, 4 ]
+
+  }
+
+]
+```
+
+3)   Create a namespace and secret for object storage credentials with below commands:
+
+```shell
+kubectl create namespace triton
+
+kubectl create secret generic minio_cred –from-literal=AWS_ACCESS_KEY_ID=<specify_access_key> --from-literal=AWS_SECRET_ACCESS_KEY=<specify_secret_access_key> -n triton
+```
+
+4) Create a deployment to host the model and check pods and services are running
```

Comment:
rephrase last part of sentence to "...and check that the pods and services are running"